### ISSUE TITLE
Finished MVP 3

### ISSUE THREAD

**Issue by wburke415** [wburke415/GarageSale#3]
Users can search for products, bid on them or buy them, and list them

**Comment by dwebster17** [comment]
Everything is looking super functional. Awesome job! Here are some quick fixes I'd like to see:

- [ ] I messed with the CSS on your spash page a bit, and got this: 

<img width="900" alt="screen shot 2018-08-17 at 12 53 22 pm" src="https://user-images.githubusercontent.com/29282483/44278621-8e995e80-a21c-11e8-93f6-a37bc187478a.png">

I think these subtle tweaks can make it look a lot more professional; CSS is extremely important, especially for a clone of eBay which has a relatively simple UI to start. Here's what's added in that screenshot:
    - on `hover` of the `splash-img-container`, I added a 1px grey border.
    - I changed the dimensions on the frontend CSS of the `img` so that it no longer stretches, and has room to breath. The CSS for that involves:
     ```css
           height: 90%;
           width: 75%;
           object-fit: cover
     ```
    - I change the font of the headers - using Times is generally rough. I simply changed `font-family` to `sans-serif` and changed the `font-weight` to `500`
    - On the `daily-deals-products` class I added margins on the sides, so it is now `margin: 40px 20px`. The issue here is that now the page overflows - can you make it so only **5** items are featured per section?
    - I reduced the `margin-top` on the price `span` 

Sounds like a lot of changes, but they'll make a big difference, and this page is your website's first impression!

One more thing to consider in the future of this project is a cron job that creates new listings every day (or resets the expiration time). It is hard for me to test out the bidding feature if every product has expired and I can't bid. This link should explain that: https://devcenter.heroku.com/articles/scheduler

Overall, awesome job and keep up the great work!!

**Comment by wburke415** [comment]
Thanks for the tips! I made most of the changes that you suggested. I decided to put the border on the image itself instead of the container though, and I was having some issues messing with the side margins for some reason so I decided to just use space-around instead of space-between and I think it worked pretty well. 

I'll look into the scheduler so I can reset expirations. If you were seeing a bunch of listings already ended, I'm pretty sure thats because I had a bug with the bidding on a couple of them and I was going through all the listings buying and bidding on them just for testing. I'm gonna reseed my database in just a moment.
